centreon/www/install/php/Update-next.php

```diff
@@ -189,7 +200,7 @@
     $errorMessage = 'Unable to update table cb_list_values';
     $pearDB->executeQuery(
         <<<SQL
-        ALTER TABLE cb_list_values DROP CONSTRAINT `fk_cb_list_values_1`
+        ALTER TABLE cb_list_values DROP CONSTRAINT IF EXISTS `fk_cb_list_values_1`
```

Comment:
DROP CONSTRAINT IF EXISTS is not possible on MySQL
You can use the method of CentreonDB `isConstraintExists` that will return you a bool

https://github.com/centreon/centreon/blob/87d573bdde30a6a5339c5c378e0f67142bce384d/centreon/www/class/centreonDB.class.php#L860